src/System.Buffers.Experimental/System/Range.cs

```diff
@@ -3,22 +3,23 @@
 
 using System.Collections;
 using System.Collections.Generic;
+using System.Text;
 
 namespace System
 {
     // TODO: consider allowing Last > First. Ennumeration will count down.
-    public struct Range : IEnumerable<int>
+    public readonly struct Range : IEnumerable<int>
     {
         public const int UnboundedFirst = Int32.MinValue;
-        public const int UnboundedLast = Int32.MaxValue;
+        public const int UnboundedEnd = Int32.MaxValue;
 
         public readonly int First;
```

Comment:
`First` should be renamed to `Start` to follow naming convention as @jcouv mentioned earlier in the previous PR: `Start` and `End`.

Reply:
Yeah, I am not thrilled with Start. It sounds like an action is about to start. That's why I originally did not use End. But I will think about the naming a bit more.

Reply:
I agree with @YohDeadfall  as `Start` seems to fit with `End`.  However, I also see @KrzysztofCwalina point.  I like `First` and `Last` better, though.

Reply:
`Begin` would be fine, but unfortunately it shouldn't be used for historical reasons since everywhere in .NET only `Start`/`End` is used for such situations in parameter names, fields, properties. Also `Begin` and `End` are used for pre-`Task` async operations in method names.